packages/runtime/src/errors/errors-list.ts

```diff
@@ -385,6 +385,12 @@ const runtimeAsaErrors = {
     message: "All of the created assets should be in creator's account",
     title: "Asset Total Error",
     description: "Asset Total Error"
+  },
+  CANNOT_CLOSE_ASSET_BY_CLAWBACK: {
+    number: 1509,
+    message: `Assetholding of account cannot be closed by clawback`,
+    title: "Close asset by clawback error",
+    description: "Cannot close asset by clawback"
```

Comment:
please add more details into description. Maybe a link to documentation.

Reply:
I couldn't find in documentation regarding this behaviour. I got to know it by asking on discord. In documentation they mention `AssetCloseTo` param in clawback transaction reference - [https://developer.algorand.org/docs/reference/transactions/?query=clawback#asset-clawback-transaction](https://developer.algorand.org/docs/reference/transactions/?query=clawback#asset-clawback-transaction), which led to confusion

Reply:
I will add a link for general asset clawback behaviour